### ISSUE TITLE
ScanElf failed with `uncaught_exception`

### ISSUE THREAD

**Issue by tuduongquyet** [target/strelka#199]
**Describe the bug**
ScanElf failed with `uncaught_exception`

**Steps to reproduce**
1. Build and run `strelka-fileshot` or `strelka-oneshot` on Ubuntu 20.04
```
./strelka-oneshot -f strelka-fileshot
```
2. Read response from server
```
{
  "file": {
    "depth": 0,
    "flavors": {
      "mime": [
        "application/x-executable"
      ],
      "yara": [
        "elf_file"
      ]
    },
    "scanners": [
      "ScanElf",
      "ScanEntropy",
      "ScanFooter",
      "ScanHash",
      "ScanHeader",
      "ScanYara"
    ],
    "size": 12777735,
    "tree": {
      "node": "7783472d-7d2a-4703-b714-b098f5365f18",
      "root": "7783472d-7d2a-4703-b714-b098f5365f18"
    }
  },
  "request": {
    "attributes": {
      "filename": "strelka-fileshot"
    },
    "client": "go-oneshot",
    "id": "7783472d-7d2a-4703-b714-b098f5365f18",
    "source": "ubuntu",
    "time": 1650964452
  },
  "scan": {
    "elf": {
      "elapsed": 0.583676,
      "flags": [
        "uncaught_exception"
      ]
    },
    "entropy": {
      "elapsed": 0.028771,
      "entropy": 6.690920985654504
    },
    "footer": {
      "elapsed": 5.7e-05,
      "footer": "me.buildVersion.str\u0000type.*\u0000runtime.textsectionmap\u0000"
    },
    "hash": {
      "elapsed": 0.177556,
      "md5": "40d131ae6f009d175c00aa12dec26895",
      "sha1": "26fb220f7450674ca63324299b28e9292b2ae73f",
      "sha256": "7cc948620272bdb96cb62ab1c94c4ae0dbcd1f261b9e35c53557773cb71b7d68",
      "ssdeep": "196608:hSsNJlt4pAgVLA7H2UINxrnK5YVKUKvKh:UQUpAgZiArrnKq4v2"
    },
    "header": {
      "elapsed": 7.7e-05,
      "header": "\u007fELF\u0002\u0001\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0002\u0000>\u0000\u0001\u0000\u0000\u0000�cF\u0000\u0000\u0000\u0000\u0000@\u0000\u0000\u0000\u0000\u0000\u0000\u0000�\u0001\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000"
    },
    "yara": {
      "elapsed": 0.035788,
      "matches": [
        "test"
      ]
    }
  }
}
```
**Expected behavior**
Elf scanner runs properly

**Release**
 - Release: [e.g. latest or 0.21.11.29]

**Comment by phutelmyer** [comment]
Thanks @tuduongquyet - I appreciate you reporting this!

**Comment by phutelmyer** [comment]
Should be fixed in #202

**Comment by tuduongquyet** [comment]
Thank you ^^
